### PR TITLE
Updated README.md and added INSTALL.md

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+You can start OpenSlides with Docker Compose. It is also possible to use Docker
+Swarm or other container orchestration systems instead of Docker Compose for
+bigger setups, but this is not described here.
+
+
+### Docker and Docker Compose
+
+So for now let's use Docker Compose. Be sure you have
+[Docker](https://docs.docker.com/engine/) and [Docker
+Compose](https://docs.docker.com/compose/) installed and the Docker daemon is
```
Comment (rrenkert) [inline INSTALL.md:14]: Do we have a minimum docker/compose version?

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+Wait until all services are available. Then run in a second terminal in the same
+directory:
+
+    $ ./openslides initial-data
+
+Now open https://localhost:8000, login with superuser credentials (default
+username and password: `superadmin`) and have fun.
+
+
+### Reach OpenSlides from outside your local machin
```
Comment (rrenkert) [inline INSTALL.md:73]: Typo: Missing "e" in machin"e"

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+    $ docker-compose stop
+
+To remove all containers run:
+
+    $ docker-compose rm
+
+To remove the database you have to remove the content of the `db-data`
+directory.
+
```
Comment (rrenkert) [inline INSTALL.md:102]: To do a deeper cleanup (networks and containers) there is 
    $ docker-compose down

I think we should add this here.

File: INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+After you have such a file remove your Docker Compose YAML file and rerun the
+`setup` command:
+
+    $ ./openslides setup --config my-config.yml .
+
+This way you get a Docker Compose YAML file which let OpenSlides listen on the
+configured custom port.
+
+You may also use  the `--force` flag in some cases which also resets secrets and
+much more. To rebuild the Docker Compose YAML file without resetting the whole
```
Comment (rrenkert) [inline INSTALL.md:125]: Do we have a more detailed description of "much more"?

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -7,51 +8,20 @@ managing and projecting agenda, motions and elections of an assembly. See
...
-
 
 ## License and authors
 
-OpenSlides is Free/Libre Open Source Software (FLOSS), and distributed
-under the MIT License, see ``LICENSE`` file. The authors of OpenSlides are
-mentioned in the ``AUTHORS`` file.
+OpenSlides is Free/Libre Open Source Software (FLOSS), and distributed under the
+MIT License, see [LICENSE file](LICENSE). The authors of OpenSlides are
+mentioned in the [AUTHORS file](AUTHORS).
```
[the remark this answers is not in the document]
Comment (emanuelschuetze) [reply README.md:27]: Done in #6524

**Pull request merged**